src/entity/learning-object/learning-object.ts

```diff
@@ -638,8 +642,10 @@ export class LearningObject {
    * @memberof LearningObject
    */
   private copyObject(object: Partial<LearningObject>): void {
-    if (object._id) {
-      this._id = object._id;
+    if (object.id) {
+      this.__id = object.id;
+    } else if (object._id) {
+      this.__id = object._id;
```

Comment:
The FullLearningObject _id is a mongo id of the objects-index collection which cannot be used to make api requests. John has a PR up that address the id issue.

Reply:
FullLearningObject is the return from clark-service*^